dev/README.md

```diff
@@ -116,17 +116,185 @@ incompatible way at any day.
 
 ## Requirements
 
-* docker
-* git
-* go
-* helm
-* jq
-* k3d
-* kubectl
-* ...
+- docker
+- git
+- go
+- helm
+- jq
+- k3d
+- kubectl
+- ...
 
 ## Run integration tests
 
     ./dev/run-integration-tests.sh
 
 This will download and prepare setup-envtest, then it will execute all the integration tests.
+
+## Running Github Actions locally
+
+Sometimes, it may be beneficial to be able to run the Github Action tests using
+the same configuration which is used remotely. To do this, you can use
+[nektos/act](https://github.com/nektos/act).
+
+### Requirements
+
+- [Docker](https://www.docker.com/)
+- [act](https://github.com/nektos/act#installation)
+
+### Installation
+
+To install `act`, please follow the [instructions on the official Github
+repository](https://github.com/nektos/act#installation).
+
+### Configuration
+
+#### Container Image
+
+Unlike Github Actions, `act` will use container images for running the tests
+locally. The container image that will be used depends on the type of the
+Github Action Runner for the specific action. You can see which images are being
+used for which runner [here](https://github.com/nektos/act#runners). Most tests require `ubuntu-latest`.
+
+The containers are available in difference sizes:
+
+- Micro
+- Medium
+- Large
+
+The default container image for Ubuntu, which is required for most actions, does
+not contain all necessary tools. They are [intentionally
+incomplete](https://github.com/nektos/act#default-runners-are-intentionally-incomplete).
+Instead, the large container image needs to be used (read on for an
+alternative).
+
+To change the container image used for Ubuntu, you will need to create a
+configuration file `$HOME/.actrc`.
+
+```shell
+-P ubuntu-latest=catthehacker/ubuntu:full-latest
+```
+
+While the medium-sized container image has a size of only 1.1GB, the large
+container image is significantly larger, approximately 40GB in size. If this is
+a concern, you can create your own container image using the following
+`Dockerfile`. This will result in a container image of about 700MB, while still
+including all the necessary tools to run both single and multi-cluster tests of
+fleet.
+
+```Dockerfile
+FROM ubuntu:22.04
+
+RUN apt update && apt upgrade -y
+RUN apt install -y wget curl git jq nodejs docker.io
```

Comment:
Please drop `docker.io` in favor of another installation method. According to the docs, this package is deprecated, and in fact does not includes the `buildx` plugin so using BuildKit does not work.

You could use the [convenience script](https://docs.docker.com/engine/install/ubuntu/#install-using-the-convenience-script), for example:
```
RUN curl -fsSL -o get_docker.sh \
            https://get.docker.com && \
        chmod 700 get_docker.sh && \
        ./get_docker.sh && rm get_docker.sh
```